tests/foreman/installer/test_installer.py

```diff
@@ -165,6 +273,19 @@ def sat_default_install(module_sat_ready_rhels):
     return sat
 
 
+@pytest.fixture(scope='module')
+def sat_default_install_fapolicyd(module_sat_ready_rhels):
```

Comment:
@rmynar `sat_default_install` and `sat_default_install_fapolicyd` are both using `module_sat_ready_rhels` won't this affect tests in some way?

Reply:
@jameerpathan111 They might be using two different satellites though cause `module_sat_ready_rhels` is spinning 2 satellites.

TBH, this is a very bad design of spinning two sats and deciding them in 2 tests. Even though you wish to run just 1 test out of it, the fixture will unnecessarily spin up 2 systems.

I am changing that in https://github.com/SatelliteQE/robottelo/pull/15948/files#diff-606dfd8ed66a37a0a150b787d8ce417b246c8d3e0aaf8b33c4d35da5189f5792R161

Reply:
If I understand correctly the original intent of this implementation, the purpose was to make testing faster. All hosts are spinned up concurrently during setup, then tests are run sequentially.
Btw i'm only changing the count of deployed hosts, not the implementation itself.